src/components/network/network-map.js

```diff
@@ -164,6 +164,24 @@ const NetworkMap = forwardRef((props, ref) => {
         }
     }
 
+    function getLabelPosition(network, line, percent, fullPath) {
```

Comment:
is this used ?

Reply:
It is.

Reply:
where ?

Reply:
didn't look at the right function, good remark, deleted.